src/components/map/layers/RoutePathLinkLayer.tsx

```diff
@@ -47,29 +47,32 @@ class RoutePathLayer extends Component<RoutePathLinkLayerProps> {
     }
 
     private renderNodes() {
+        const openPopup = (node: INode) => () => {
```

Comment:
Would it be possible to place this outside of the function scope? Declaring functions inside the function forces us to recreate the function, every time the parent function runs